TestPlaybooks/playbook-IntSights_Test.yml

```diff
@@ -458,6 +458,47 @@ tasks:
           "y": 1770
         }
       }
+    note: false
+    timertriggers: []
+  "27":
+    id: "27"
+    taskid: 9ee08e2b-cf55-4d7e-89e5-f0e1a274e8f9
+    type: regular
+    task:
+      id: 9ee08e2b-cf55-4d7e-89e5-f0e1a274e8f9
+      version: -1
+      name: intsights-get-alerts time-delta
+      script: '|||intsights-get-alerts'
```

Comment:
You should verify the results of the command in the test

Reply:
Yeah so the problem is that this new feature of time-delta does not allow us to test this properly as we don't have alerts for about a year, so we thought it would be best to just make sure that if you pass this argument that nothing fails

Reply:
Oh, so in that case we just need mocks :)

Reply:
in a month (;